test/token/erc20/preset/ImmutableERC20.t.sol

```diff
@@ -68,13 +68,12 @@ contract ImmutableERC20Test is Test {
     }
 
     function testBurn() public {
-        address from = makeAddr("from");
         uint256 amount = 100;
         vm.prank(minter);
-        erc20.mint(from, amount);
-        assertEq(erc20.balanceOf(from), 100);
-        erc20.burn(from, amount);
-        assertEq(erc20.balanceOf(from), 0);
+        erc20.mint(minter, amount);
```

Comment:
It would be good to mint to a different address, to show that the minter (a special account) isn't able to burn